test/read.test.js

```diff
@@ -14,40 +15,61 @@ describe('read', () => {
 
   ;[
     [
-      'on a FTP server with anonymous user',
-      () => new FtpServer(),
-      {}
-    ],
-    [
+      'on a FTP server with anonymous user', () => new FtpServer(), {}], [
       'on a FTP server with username/password',
       () => new FtpServer({ user: 'test', password: '1234' }),
-      {}
-    ],
-    [
-      'on a SFTP server with anonymous user',
-      () => new SftpServer(),
-      {}
-    ],
-    [
+      {}], [
```

Comment:
Can you configure your code formatter?

Here it's formatting in a way where it's hard to read, which is not great.
The previous way was easy to read and was consistent.

Reply:
Sure!

Reply:
I recommend [husky](https://npm.im/husky) with a pre-commit hook. Combined with [lint-staged](https://npm.im/lint-staged) it provides a good way to ensure consistent code-style and formatting on new and modified files